en/docs/get-started/try-out-flow.md

```diff
@@ -8,68 +8,43 @@ Service (AIS) as a sample. For more information, see the [Try out](../try-out/ac
-              layer security testing purposes.
-            - Use the [signing certificate](../../assets/attachments/signing-certs/obsigning.pem) and
-              [signing private keys](../../assets/attachments/signing-certs/obsigning.key) for signing purposes.
-
-    ``` tab='Format'
-    
-    {
-    "alg": "<The algorithm used for signing.>",
-    "kid": "<The thumbprint of the certificate.>",
-    "typ": "JWT"
-    }
-    {
-    "iss": "<This is the issuer of the token. For example, client ID of your application>",
-    "sub": "<This is the subject identifier of the issuer. For example, client ID of your application>",
-    "exp": <This is the epoch time of the token expiration date/time>,
-    "iat": <This is the epoch time of the token issuance date/time>,
-    "jti": "<This is an incremental unique value>",
-    "aud": "<This is the audience that the ID token is intended for. For example, https://<IS_HOST>:9446/oauth2/token>"
-    }
-   
-    <signature: The client assertion must be signed using the private key of the application certificate.>
-
-    ```
-    
-    ``` tab='Sample'
-    eyJraWQiOiJXX1RjblFWY0hBeTIwcTh6Q01jZEJ5cm9vdHciLCJhbGciOiJQUzI1NiJ9.eyJzdWIiOiJQU0RHQi1PQi1Vbmtub3duMDAxNTgwMDAwMUhRUXJaQUFYIiwiYXVkIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6OTQ0Ni9vYXV0aDIvdG9rZW4iLCJpc3MiOiJQU0RHQi1PQi1Vbmtub3duMDAxNTgwMDAwMUhRUXJaQUFYIiwiZXhwIjoxODI3NjY5MzMyLCJpYXQiOjE4Mjc2NTkzMzIsImp0aSI6IjE0MzIyNDM2MzQzNDM1NDQ1In0.qUq9q_Qa5eiVW5C6QzMvB1sX9Ttwz0Db8c2wmRXyrUWDpeoaolUYT_Diu1o33R4U4MME3nBMCdl0wQ1AVnuzjgV6s3TLcyxlphcoGXYVwOLsQBfbLKTzGiz10UORb3WQc9BwxhZVPDWyFXGlqUNwjPbaUslWoal9KMsbnXlBFKQd8GWjhS-kXHn66kAHwH-7DLZ_Z7D01oW2aFon5sWBZfKD_t9NeQJ9gdPs45ermSM45FixlKXkPPXiIq-_w5Hw1Zw_lEW6fVpWCS6IRz5pBtpHO8s_KESjxuPb1dzrV31AZC7BplWeaRRC5UslObbejw35P5v9CQqJR5Uc7_mX0Q
-    ```
-
-2. Run the following cURL command in a command prompt to generate the access token. Update the placeholders with relevant values.
-``` curl
-curl -X POST \
-https://<IS_HOST>:9446/oauth2/token \
---cert <TRANSPORT_PUBLIC_KEY_FILE_PATH> --key <TRANSPORT_PRIVATE_KEY_FILE_PATH> \
--d 'grant_type=client_credentials&scope=accounts%20openid&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer&client_assertion=<CLIENT_ASSERTION_JWT>&redirect_uri=<REDIRECT_URI>&client_id=<CLIENT_ID>'
-```
-
-3. Upon successful token generation, you can obtain a token as follows:
+        - Use the [transport private key](../../assets/attachments/transport-certs/obtransport.key) and
+        [transport public certificate](../../assets/attachments/transport-certs/obtransport.pem) for Transport
+        layer security testing purposes.
+        - Use the [signing certificate](../../assets/attachments/signing-certs/obsigning.pem) and
+        [signing private keys](../../assets/attachments/signing-certs/obsigning.key) for signing purposes.
+
+   ``` curl
+   curl -X POST \
+   https://localhost:9446/oauth2/token \
```

Comment:
Shouldn't we use the placeholder here?

Reply:
Fixed in https://github.com/wso2/docs-ob-toolkit-berlin/pull/83/commits/a33b250d92c0799adb3b95afaa39531ea247d802